### PR TITLE
Fix MapAccessExprTest test case

### PR DESCRIPTION
**Pull request by daneshk**
## Purpose
Fix MapAccessExprTest test case

### CODE REVIEW

File: tests/ballerina-unit-test/src/test/resources/test-src/types/map/map-value.bal [hunk 1]
```diff
@@ -13,7 +13,7 @@ function testMapWithMap() returns (string){
     return list2String;
 }
 
-function testMapWithAnyValue() returns (int){
+function testMapWithAnyValue() returns (int|error){
```
Comment (VijithaEkanayake) [inline tests/ballerina-unit-test/src/test/resources/test-src/types/map/map-value.bal]: Can we fix the formatting issue as well?

Comment (daneshk) [reply tests/ballerina-unit-test/src/test/resources/test-src/types/map/map-value.bal]: Fixed in 42ca94a

File: tests/ballerina-unit-test/src/test/resources/test-src/io/csv_io.bal [hunk 2]
```diff
@@ -66,5 +66,8 @@ function getTable(string filePath, string encoding, io:Separator fieldSeperator)
         return total;
     } else if (tableResult is error) {
         return tableResult;
+    } else {
+        error e = error("Record channel not initialized properly");
```
Comment (VijithaEkanayake) [inline tests/ballerina-unit-test/src/test/resources/test-src/io/csv_io.bal]: Should this go to error details?
ex:
error e = error(IO_ERROR_CODE, { message : "Record channel not initialized properly" });

Comment (daneshk) [reply tests/ballerina-unit-test/src/test/resources/test-src/io/csv_io.bal]: Fixed in 712af3e

**Pull request merged**